### DISCUSSION TITLE
第5节课笔记(15班)

### DISCUSSION THREAD

**Discussion by ironartisan** [InternLM/Tutorial#40] [笔记]
请在下方提交笔记链接

**Comment by szhowardhuang** [comment]
第5节课内容依旧不少，除了刚开始的环境搭建，后续学习过程顺利。

上传pdf
[LM Deploy.pdf](https://github.com/InternLM/tutorial/files/13931044/LM.Deploy.pdf)

**Comment by scchy** [comment]
第5节课笔记:https://github.com/scchy/My_Learn/blob/master/LLM_prime/5_InternLM_Imdeploy_doc.md

**Comment by woodx9** [comment]
请参考笔记部分记录[飞书链接](https://gxloh9mfvxh.feishu.cn/wiki/OvLxw1ClfinJAZkuDwzcpd9ZnZb?from=from_copylink)

**Comment by Mrguanglei** [comment]
https://gzr5xfkph4.feishu.cn/docx/QDSPdw0OKow6Rzx3lapcUNlWn5b?from=from_copylink

**Comment by IyGnahz** [comment]
笔记+作业 https://g6xuom3zbh.feishu.cn/docx/Ym96d6IYeocm3KxDOCnc1s1Lnac

**Comment by dqyydq** [comment]
https://blog.csdn.net/qq_52410386/article/details/135669248?spm=1001.2014.3001.5501

**Comment by lizhunkg** [comment]
第五课笔记作业地址： https://note.youdao.com/s/2UprW40e

**Comment by seienn** [comment]
https://gvxxiiwsufy.feishu.cn/wiki/VgvxwJcy8i3LoTkox3RcWqiGngY?from=from_copylink

**Comment by bblgbr** [comment]
第五节课笔记： https://github.com/bblgbr/play_llm/blob/main/InterLM_Class/05%20LMDeploy%E5%A4%A7%E6%A8%A1%E5%9E%8B%E9%83%A8%E7%BD%B2.md

**Comment by sokolo05** [comment]
第五节课程笔记：
https://github.com/sokolo05/Scholar_PuYu/blob/main/02.%E8%AF%BE%E7%A8%8B%E7%AC%94%E8%AE%B0/%E7%AC%AC5%E8%8A%82_%E7%AC%94%E8%AE%B0.md

**Comment by ziptoam** [comment]
https://note.youdao.com/s/D5x51QmK

**Comment by yabgdsajghsj** [comment]
https://note.youdao.com/s/VUNBK40g

**Comment by Mrguanglei** [comment]
https://mqf0lvyk616.feishu.cn/wiki/EuvTwGfNii71LWkQQibcPdaDnHb?from=from_copylink

**Comment by NANDSS1** [comment]
https://www.yuque.com/nandss/gyyg9p/egq168ld87gf8u4c?singleDoc# 《lesson 5 LMDeploy 大模型量化部署实践笔记》

**Comment by g29times** [comment]
https://novagen.notion.site/5-7e03e39db3a94e5490ac6ec3281c9406?pvs=4
